openweather.js

```diff
@@ -28,6 +28,7 @@ const subscriptions = [
     rain: 4,
     weathercode : 200 
 */
+const currentTempPath = "environment.outside.temperature";
```

Comment:
This path may conflict with data exposed by other plugins like BME280 or RUUVI tag

Reply:
The SignalK server can resolve multiple paths from different senders.  This is preferable to having different paths for the same value.

For example, multiple GPS units on the network may all provide lat/lon data.

Reply:
Understood and acknowledged - however, this is still a different situation to capturing gps with multiple devices onboard, ie. local sensors capturing significantly different data than returned from the API which can only be an approxmiate median value (if any when in an off-shore location). Anyone following that data (eg. by uploading to a timeseries on influx or for feeding into predictions) will then have to differentiate per data source if the data is trustworthy. Hence, the additional path would at least have to be configurable. In addition need to make clear whether or not to treat `pressure` and `humidity` in the same way.